### PR TITLE
OTWO-6684 add recent projects in home page

### PR DESCRIPTION
**Pull request by Niharika1117**

### CODE REVIEW

File: app/decorators/home_decorator.rb [hunk 1]
```diff
@@ -47,4 +47,8 @@ def repository_count
       Enlistment.connection.execute('select count(*) from repositories').values[0][0].to_i
     end
   end
+
+  def most_recent_projects
+    Project.where('created_at > ?', 30.days.ago).limit(10)
```
Comment (sig-krasnick) [inline app/decorators/home_decorator.rb:52]: order by created_at date descending

File: config/locales/home.en.yml [hunk 2]
```diff
@@ -14,6 +14,7 @@ en:
       popular_projects: 'Most Popular Projects'
       active_projects: 'Most Active Projects'
       active_contributors: 'Most Active Contributors'
+      recent_projects: 'Most Recent Projects'
```
Comment (sig-krasnick) [inline config/locales/home.en.yml:17]: Let's change this to 'Recently Added Projects'.

File: app/decorators/home_decorator.rb (hunk 1 above)
Comment (Niharika1117) [reply app/decorators/home_decorator.rb]: done

File: config/locales/home.en.yml (hunk 2 above)
Comment (Niharika1117) [reply config/locales/home.en.yml]: done

**Pull request merged**